### DISCUSSION TITLE
A solution to fix the overlap between the "selected node" style and the background color of the node.

### DISCUSSION THREAD

**Discussion by aaa1386** [freeplane/freeplane#1604] [Ideas]
The background color of the node is meaningful for the user, for example, sometimes the user changes the background color of important nodes to green.

There is a problem when the green node is selected the background color changes to the "selected node" style color. The user does not know what the real color of the node is?

To fix the overlap, you can adjust the style of the selected node as shown below.
![image](https://github.com/freeplane/freeplane/assets/38586259/1e69c8ae-90eb-4c11-b033-3610071da7bf)

The cloud mode should be activated for the "selected node" style.
 
In this case the background color of the node does not change when it is selected.
The only problem is that the cloud state of the selected node and its sub-branches are all clouded.
If software developers add an option where there are two modes to choose from:
1- clouding the "selected node"
2- clouding the "selected node and all its children"

By doing this, the user has more options and the problem of overlapping between the background color of the "selected node" style and the background color of the node is solved.

**Comment by aaa1386** [comment]
Providing two choices for cloud mode doesn't hurt the software but makes it more complete.

**Comment by dpolivaev** [comment]
Instead of using a cloud, you can modify the color, the width, and the shape of the border.

**Reply by aaa1386** [reply]
I'm having trouble using a border for the "selected node" style.
Overrides other defined styles.
Watch the video below:


https://github.com/freeplane/freeplane/assets/38586259/bca47d64-57bf-4da6-8c5e-391a80ad0c22

**Reply by aaa1386** [reply]
The correct operation happens only in one case when the default "line width" of the software is 1 Pt.
Pay attention to the following video:



https://github.com/freeplane/freeplane/assets/38586259/eda092cf-0519-472e-b820-5b0fe4dcea57

**Reply by dpolivaev** [reply]
Yes, you should wisely decide which properties are modified for selected nodes and which not.

**Comment by sifran-github** [comment]
check the option: preferences -> defaults -> selection colors -> display selected nodes in bubbles.
it displays a bubble on top of the selected node, instead of changing its color.

this is the result:
![image](https://github.com/freeplane/freeplane/assets/5730355/7e409e0c-ea9a-425d-94bb-08bdeade3b4f)

even if i change the color to the same bubble color, it is still recognizable:
![image](https://github.com/freeplane/freeplane/assets/5730355/8fcf1fc7-ffc9-4141-b1fc-d11fab702376)

I always use this option.
I think that this should be the default, its better :)

**Reply by aaa1386** [reply]
Thank you .
It was good advice.
I also like this method more. In fact, I was looking for something that is completely independent of the node style.
The only thing is, it would be great if the software developer would put an option to increase or decrease the width of the border line or  border like a cloud or other settings.
Freeplane is the best✅✅✅

**Reply by aaa1386** [reply]
What is the use of "Standard selected node color (Fallback)"?
